src/Config_Command.php

```diff
@@ -222,9 +222,40 @@ public function path() {
 	 *     | SECURE_AUTH_KEY  | iO-z!_m--YH$Tx2tf/&V,YW*13Z_HiRLqi)d?$o-tMdY+82pK$`T.NYW~iTLW;xp | constant |
 	 *     +------------------+------------------------------------------------------------------+----------+
 	 *
+	 *     # List only database user and password from wp-config.php file.
+	 *     $ wp config list DB_USER DB_PASSWORD --strict
+	 *     +------------------+-------+----------+
+	 *     | key              | value | type     |
+	 *     +------------------+-------+----------+
+	 *     | DB_USER          | root  | constant |
+	 *     | DB_PASSWORD      | root  | constant |
+	 *     +------------------+-------+----------+
+	 *
+	 *     # List all salts from wp-config.php file.
+	 *     $ wp config list _SALT
+	 *     +------------------+------------------------------------------------------------------+----------+
+	 *     | key              | value                                                            | type     |
+	 *     +------------------+------------------------------------------------------------------+----------+
+	 *     | AUTH_SALT        | n:]Xditk+_7>Qi=>BmtZHiH-6/Ecrvl(V5ceeGP:{>?;BT^=[B3-0>,~F5z$(+Q$ | constant |
+	 *     | SECURE_AUTH_SALT | ?Z/p|XhDw3w}?c.z%|+BAr|(Iv*H%%U+Du&kKR y?cJOYyRVRBeB[2zF-`(>+LCC | constant |
+	 *     | LOGGED_IN_SALT   | +$@(1{b~Z~s}Cs>8Y]6[m6~TnoCDpE>O%e75u}&6kUH!>q:7uM4lxbB6[1pa_X,q | constant |
+	 *     | NONCE_SALT       | _x+F li|QL?0OSQns1_JZ{|Ix3Jleox-71km/gifnyz8kmo=w-;@AE8W,(fP<N}2 | constant |
+	 *     +------------------+------------------------------------------------------------------+----------+
+	 *
 	 * @when before_wp_load
+	 * @subcommand list
 	 */
-	public function get( $_, $assoc_args ) {
+	public function list_( $args, $assoc_args ) {
+ 	    $path = Utils\locate_wp_config();
```

Comment:
Another trivial one, there's a space at the beginning of the line here before the tab, and in 4 other places `/^ \t/`

Reply:
Fixed. Copy/paste issue because I copied from a previous implementation I had already previously done.